include/libtorrent/entry.hpp

```diff
@@ -95,8 +95,30 @@ namespace libtorrent
 			bool operator()(T1 const& rhs, T2 const& lhs) const
 			{ return rhs < lhs; }
 		};
-	};
+
+		template<class T>
+		class map_string : public std::map<std::string, T, aux::strview_less>
+		{
+		};
+#else
+		template<class T>
```

Comment:
ok, i'll wait till appveyor and travis finish their jobs and try to remove condition